docs/reference/feature-servers/go-feature-retrieval.md

```diff
@@ -0,0 +1,36 @@
+# Go-based Feature Retrieval
+
+## Overview
+
+The Go Feature Retrieval component is a Go implementation of the core feature serving logic, embedded in the Python SDK. It supports retrieval of feature references, feature services, and on demand feature views, and can be used either through the Python SDK or the [Python feature server](local-feature-server.md).
+
+Currently, this component only supports online serving and does not have an offline component including APIs to create feast feature repositories or apply configuration to the registry to facilitate online materialization. It also does not expose its own dedicated cli to perform feast actions. Furthermore, this component is only meant to expose an online serving API that can be called through the python SDK to facilitate faster online feature retrieval.
+
+The Go Feature Retrieval component currently only supports Redis and Sqlite as online stores; support for other online stores will be added soon. Initial benchmarks indicate that it is significantly faster than the Python feature server for online feature retrieval. We plan to release a more comprehensive set of benchmarks. For more details, see the [RFC](https://docs.google.com/document/d/1Lgqv6eWYFJgQ7LA_jNeTh8NzOPhqI9kGTeyESRpNHnE).
+
+## Installation
+
+To install, run `export COMPILE_GO=true` before you run `pip install feast`.
```

Comment:
This is not true - if you're using macOS or linux, x86, python 3.7-3.10, you'll get the version of feast installed with the go library anyway.

Reply:
Hm, in setup.py the build commands for compiling go and also building go are only enabled if we are using COMPILE_GO=true. I don't think we run the compilation without the flag.

Reply:
Confirmed it's precompiled and fixed.